onboarding-website/src/App.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <header class="header">
+    <navbarComponent />
+  </header>
+  <main class="main">
+    <aside class="aside">
+      <asideComponent @filter="filter" />
+    </aside>
+    <section class="section">
+      <sectionComponent :key="filterCriteria" :filterCriteria="filterCriteria" />
+    </section>
+  </main>
+</template>
+
+<script setup>
+  import navbarComponent from '@/components/navbarComponent.vue';
+  import asideComponent from '@/components/asideComponent.vue';
+  import sectionComponent from './components/sectionComponent.vue';
+  import { useTaskStore } from './stores/taskData';
+  import { useCategoryStore } from './stores/category';
+  
+  import { ref } from 'vue';
+  
+  const filterCriteria = ref('');
+  function filter(value) {
+    filterCriteria.value = value.toUpperCase();
+  }
+  
+  if (localStorage.getItem('taskStorage') && useTaskStore().data.task.length === 0) {
```

Comment:
I guess comparison with 0 is not needed.

Reply:
resolve